src/opengl/ogl_shader.c

```diff
@@ -282,6 +285,9 @@ static bool glsl_set_shader_sampler(ALLEGRO_SHADER *shader,
 
    texture = bitmap ? al_get_opengl_texture(bitmap) : 0;
    glBindTexture(GL_TEXTURE_2D, texture);
+   
+   if (_al_bitmap_wrap_changed(bitmap))
```

Comment:
There's one more place for this in ogl_draw.c:357

Reply:
Done.